eva_sub_cli/jinja_templates/text/report.txt

```diff
@@ -23,37 +23,56 @@ METADATA VALIDATION RESULTS
 Ensures that required fields are present and values are formatted correctly.
 For requirements, please refer to the EVA website (https://www.ebi.ac.uk/eva/?Submit-Data).
 
-{{ metadata_validation_report(validation_results) }}
-
+{% set status = validation_results.get('metadata_check', {}).get('pass', '') %}
+{% if status == "Process not run yet" %}
+    This validation is yet to run
```

Comment:
I'd prefer "Process not run yet" on both reports - clearer (to me) and matches the "Process failed" message

Also to keep things consistent we can use the pause symbol in the text report too, just need its unicode codepoint

Reply:
updated